lib/schemas/api/results.api.hearingResultedResponse.json

```diff
@@ -2,7 +2,15 @@
     "$schema": "http://json-schema.org/draft-04/schema#",
     "id": "http://justice.gov.uk/results/courts/api/hearingResultedResponse.json",
     "type": "object",
-    "allOf": [{
-        "$ref": "../../external/global/apiHearing.json"
-    }]
-}
+    "properties": {
+        "hearing": {
+            "description": "The details of the resulted hearing, its case details and the results",
+            "$ref": "../../external/global/apiHearing.json#"
+        },
+        "sharedTime": {
+            "description": "The date and time that teh results were shared by HMCTS",
+            "format": "date-time"
+        }
+    },
+    "additionalProperties": false
+}
```

Comment:
Github is flagging a missing newline at the end of this file

Reply:
Thanks, I did not add it because it was an HMCTS provided change